robottelo.properties.sample

```diff
@@ -303,6 +303,31 @@ ssh_key=
 # 'semanage port -a -t websm_port_t -p tcp <port-range>'
 port_range=9091, 14999
 
+# [ec2]
+# AWS EC2 to be added as a compute resource.
+# Access key ID of the IAM user
+# access_key=
```

Comment:
@san7ket , @ldjebran and @rochacbruno , adding support for passing the `access_key` and `secret_key` in the robottelo.properties file in the `robottelo-ci` project.